### DISCUSSION TITLE
addKaboom is undocumented

### DISCUSSION THREAD

**Discussion by triptych** [replit/kaboom#648] [Ideas]
addKaboom is used in many demos, but it is not documented - and only via digging into the sourcecode can you find it. It seems like this would be a source of frustration for a person learning Kaboom to see a seemingly custom method that is nowhere in the code, and not in the API so they are left wondering what they "missed" to understand how Kaboom works. 

Please document all kaboom methods on the website - especially ones used in examples and tutorial demos.

**Comment by lajbel** [comment]
addKaboom it's already documented in the next version.
